src/Vehicle/Vehicle.cc

```diff
@@ -3152,7 +3152,10 @@ void Vehicle::setSoloFirmware(bool soloFirmware)
 
 void Vehicle::motorTest(int motor, int percent, int timeoutSecs)
 {
-    sendMavCommand(_defaultComponentId, MAV_CMD_DO_MOTOR_TEST, true, motor, MOTOR_TEST_THROTTLE_PERCENT, percent, timeoutSecs, 0, MOTOR_TEST_ORDER_BOARD);
+    // Sub requires periodic motortest messages, and we send them at a high rate to improve responsiveness.
+    // On higher latency connection, `showError=true` caused lots of "Unable to send command: Waiting on previous response to same command." errors.
+    bool showError = !sub();
```

Comment:
This isn't correct. This is an ArduSub problem not a "sub" problem isn't it?

Reply:
Right. What do you think of just exposing showError on the signature `void Vehicle::motorTest(int motor, int percent, int timeoutSecs, bool showError)` and letting qml decide it?

Reply:
That sounds fine.

Reply:
ok, updated!